lib/tasks/remove_classroom_users_without_github_org_access.rake

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Checks if classroom admins have access to the associated GitHub organization. If not
+# the admin is removed from the classroom
+task :remove_classroom_users_without_github_org_access do
+  User.find_in_batches(batch_size: 250).with_index do |group, batch|
+    puts "Processing batch ##{batch}"
+    group.each do |user|
+      github_org_ids = user.organizations.pluck(:github_id).uniq
+      next unless github_org_ids.any?
+
+      github_org_ids.each do |gh_id|
+        github_org = GitHubOrganization.new(user.github_client, gh_id)
+        next if github_org.admin?(user.github_login)
```

Comment:
Would love a double check on this line. If this returns true, I want to move on to the next iteration. I only want to move forward to the payload line if `github_org.admin?(user.github_login)` returns false. I'm 99% sure I've written it right, but, you can never be too sure!

Reply:
Is it true that admins cannot have their access to an organization revoked? I would think we would want to respect the right of an organization to restrict access to their repositories from anyone they want.